src/ktn_code.erl

```diff
@@ -57,17 +58,28 @@
 %% @doc If the beam was not compiled with debug_info
 %%      the code generated by this function will look really ugly
 %% @end
--spec beam_to_erl(string(), string()) -> ok.
+-spec beam_to_string(beam_lib:beam()) -> {ok, string()} | {error, beam_lib, term()}.
```

Comment:
Line 61 is too long: -spec beam_to_string(beam_lib:beam()) -> {ok, string()} | {error, beam_lib, term()}..